doc/ale-zig.txt

```diff
@@ -7,6 +7,14 @@ Integration Information
 
   Currently, the only supported linter for zig is zls.
 
```

Comment:
Add a section separator here so the help index you added to the doc/ale.txt can properly jump here. The ale-zig-zigfmt tag must match the one in the index

```
===============================================================================
zigfmt                                                              *ale-zig-zigfmt*
```

Reply:
Done